diff --git a/node_modules/next/dist/esm/server/lib/incremental-cache/index.js b/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
index a5d78f3..4726a50 100644
--- a/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
+++ b/node_modules/next/dist/esm/server/lib/incremental-cache/index.js
@@ -242,9 +242,9 @@ export class IncrementalCache {
     async set(pathname, data, revalidateSeconds, fetchCache, fetchUrl, fetchIdx) {
         if (this.dev && !fetchCache) return;
         // fetchCache has upper limit of 2MB per-entry currently
-        if (fetchCache && JSON.stringify(data).length > 2 * 1024 * 1024) {
+        if (fetchCache && JSON.stringify(data).length > 10 * 1024 * 1024) {
             if (this.dev) {
-                throw new Error(`fetch for over 2MB of data can not be cached`);
+                throw new Error(`fetch for over 10MB of data can not be cached`);
             }
             return;
         }
diff --git a/node_modules/next/dist/server/lib/incremental-cache/index.js b/node_modules/next/dist/server/lib/incremental-cache/index.js
index d1e7a59..258ad0f 100644
--- a/node_modules/next/dist/server/lib/incremental-cache/index.js
+++ b/node_modules/next/dist/server/lib/incremental-cache/index.js
@@ -269,9 +269,9 @@ class IncrementalCache {
     async set(pathname, data, revalidateSeconds, fetchCache, fetchUrl, fetchIdx) {
         if (this.dev && !fetchCache) return;
         // fetchCache has upper limit of 2MB per-entry currently
-        if (fetchCache && JSON.stringify(data).length > 2 * 1024 * 1024) {
+        if (fetchCache && JSON.stringify(data).length > 10 * 1024 * 1024) {
             if (this.dev) {
-                throw new Error(`fetch for over 2MB of data can not be cached`);
+                throw new Error(`fetch for over 10MB of data can not be cached`);
             }
             return;
         }
